src/components/IssueUI/WhoopsStatusUI/index.tsx

```diff
@@ -3,7 +3,7 @@ import { useSelector } from 'react-redux';
 import clsx from 'clsx';
 
 import RequestWrapper from 'pages/Bridge/RequestWrapper';
-import PriceInfo from 'pages/Bridge/PriceInfo';
+import PriceInfo from 'components/PriceInfo';
```

Comment:
I moved `PriceInfo` to a more generic place since it's used in many places and not limited to `Bridge` page.